src/app/app.module.ts

```diff
@@ -0,0 +1,32 @@
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+import { CommentsComponent } from './comments/comments.component';
+import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+
+import { ReactiveFormsModule } from '@angular/forms';
+import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';
+
+import { HttpClientModule } from '@angular/common/http';
+import { LoginComponent } from './login/login.component';
```

Comment:
Chú ý import theo thứ tự thư viện cũng như A-> Z
```suggestion
import { NgModule } from '@angular/core';
import { HttpClientModule } from '@angular/common/http';
import { ReactiveFormsModule } from '@angular/forms';
import { BrowserModule } from '@angular/platform-browser';

import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
import { FontAwesomeModule } from '@fortawesome/angular-fontawesome';

import { AppRoutingModule } from './app-routing.module';
import { AppComponent } from './app.component';
import { CommentsComponent } from './comments/comments.component';
import { LoginComponent } from './login/login.component';
```

Ngoài ra `FontAwesomeModule`  với `NgbModule` dùng làm gì?
`ReactiveFormsModule` có cần thiết không?